.env.example

```diff
@@ -3,3 +3,5 @@ NORDVPN_TOKEN=
 NORDVPN_ENDPOINT=San_Francisco
 NORDVPN_TECHNOLOGY=openvpn
 NORDVPN_OPENVPN_PROTOCOL=tcp
+HOSTNAME=tailnord-tailscale
+SUBNET_PREFIX=10.1.1
```

Comment:
I don't find this particularly elegant. My preference is to have some tool to do subnet arithmetic, but I don't want to over-complicate things.